### PR TITLE
16.0 account ux improvements poma

### PR DESCRIPTION
**Pull request by poma-odoo**
Description of the issue/feature this PR addresses:
Remove quick create from the places that is either not needed or not possible

Task #2850996
Related Enterprise PR: https://github.com/odoo/enterprise/pull/32206

Replaces: https://github.com/odoo/odoo/pull/98323

--
I confirm I have signed the CLA and read the PR guidelines at [www.odoo.com/submit-pr](http://www.odoo.com/submit-pr)

### CODE REVIEW

File: addons/account/views/account_account_views.xml [hunk 1]
```diff
@@ -60,7 +60,7 @@
                                 <group>
                                     <group>
                                         <field name="account_type" widget="account_type_selection"/>
-                                        <field name="tax_ids" widget="many2many_tags" domain="[('company_id','=',company_id)]" attrs="{'invisible': [('internal_group', '=', 'off_balance')]}"/>
+                                        <field name="tax_ids" widget="many2many_tags" domain="[('company_id','=',company_id)]" attrs="{'invisible': [('internal_group', '=', 'off_balance')]}" options="{'no_quick_create': True}"/>
```
Comment (fmdl) [inline addons/account/views/account_account_views.xml:63]: no_create should be better. (It is not right place to create taxes, taxes are not created every days :-) )

Comment (poma-odoo) [reply addons/account/views/account_account_views.xml:63]: Hi, it is reasonable but, this is not the goal of this task

Comment (jco-odoo) [reply addons/account/views/account_account_views.xml:63]: @fmdl It might still be in handy to create taxes in this case.  (Maybe you are setting up everything)  But the thought crossed my mind as well.

Comment (fmdl) [reply addons/account/views/account_account_views.xml:63]: @jco-odoo @poma-odoo field account_id have already `'no_create':True` https://github.com/odoo/odoo/blob/15.0/addons/account/views/account_move_views.xml#L203, it should be logical that tax are the same behavior.

on all other accounting software it is not possible to create tax on the fly.

Can you do the same things for sale and purchase ?

Comment (poma-odoo) [reply addons/account/views/account_account_views.xml:63]: I think we need to keep the things odoo style everywhere, so being able to create is the standard behavior of Odoo and I think we should keep it. Bit Quick create should be removed to prevent accidental creation of meaningless records (where name_create or creating with only name is not possible). Also, in the cases that it action of quick create is opening a form, it does the same thing as create and edit, so we don't need to options that perform the same action.
About sales and purchases, they heave dedicated teams, it is out of my scope of work.

Comment (jco-odoo) [reply addons/account/views/account_account_views.xml:63]: For sale order, purchase orders and invoices it makes sense to avoid the create all together because they are daily usage, but these are more config screens.

Comment (fmdl) [reply addons/account/views/account_account_views.xml:63]: @poma-odoo I don't understand your answer (maybe we said the same things).
Create a taxe is a complexe thing. And I should be create only in the taxes menu (like accounts).

**Comment by jco-odoo** [comment]
@robodoo r+

**Comment by poma-odoo** [comment]
replaced by https://github.com/odoo/odoo/pull/104633 , mergebot blocked this one
